wg-welcome/templates/project-specific-help.md

```diff
@@ -0,0 +1,31 @@
+# Question about a specific project
+
+## Question
+
+Sometimes people post to dev@community with a question about a specific
+Apache project. We are not the place for that question, and we need to
+gently redirect, while offering a little additional information about
+how the ASF works.
+
+## Answer
+
+Hi, _____, thanks for your interest in Apache.
```

Comment:
Two comments here:

1. Maybe we can change the placeholder as parameters so such email can even be generated with parameters set.